templates/main/fixers.html.twig

```diff
@@ -0,0 +1,33 @@
+{% block form_row %}
+    <div class="form__fixes-checkbox u-my(half) o-grid__col">
+        {{ block('form_errors') }}
+        {{ block('form_label') }}
+    </div>
+{% endblock %}
+
+{% block form_label %}
+    <label for="typo_fixers_{{ name }}">
+        <div class="form__fixes-checkbox-inner js-box
+        u-bgc(orangegradient) u-p(half) u-c(white) u-h(100) u-fz(input) u-br(5)
+        u-d(flex) u-fxd(column) u-jc(space-between) u-ai(stretch) u-cur(pointer)">
+
+            <div class="form__label-name u-bb(white)">{{ value }}</div>
+            <div class="form__label-desc u-pt(half)">{{ label }}</div>
+
+            {{ block('form_widget') }}
+
+        </div>
+    </label>
+{% endblock %}
+
+{% block form_widget %}
+    <input type="checkbox" class="js-checkbox u-my(half) u-as(center)" id="typo_fixers_{{ name }}" value="{{ value }}" name="typo_fixer[fixers][]"{% if checked %} checked="checked"{% endif %}>
+{% endblock %}
+
+{% block form_errors %}
+    {%- if errors is defined -%}
+        {%- for error in errors -%}
+            <div class="error-message u-c(error) u-fz(h5) u-bor(error) u-d(inline-block) u-py(half) u-px(base) u-br(5) u-mb(double)">{{ error.message }}</div>
+        {%- endfor -%}
+    {%- endif %}
+{% endblock %}
```

Comment:
I think this is not required, since there is already a form_theme on the global form.

Reply:
I may have done it wrong but if I leave the block empty the errors don't show up anymore